src/simulation/m_acoustic_src.fpp

```diff
@@ -171,6 +173,18 @@ contains
 
             num_points = source_spatials_num_points(ai) ! Use scalar to force firstprivate to prevent GPU bug
 
+            call random_number(phi_rn(1:100))
+            call s_mpi_send_random_number(phi_rn)
+            sum_BB = 0d0
+
+            !$acc loop
```

Comment:
Confused about this. should it be parallel loop? sequential loop? this is ambiguous